OpenRA.Mods.Common/Traits/Air/AttackAircraft.cs

```diff
@@ -24,8 +24,8 @@ public class AttackAircraftInfo : AttackFollowInfo, Requires<AircraftInfo>
 		[Desc("Attack behavior. Currently supported types are Strafe (default) and Hover.")]
 		public readonly AirAttackType AttackType = AirAttackType.Strafe;
 
-		[Desc("Delay, in game ticks, before strafing aircraft turns to attack.")]
-		public readonly int AttackTurnDelay = 50;
+		[Desc("Distance the strafing aircraft makes to a target before turning for another pass. When set to WDist.Zero this defaults to the maximum armament range.")]
+		public readonly WDist AttackRunLength = WDist.Zero;
```

Comment:
Thinking about this more, woudn't it something like `StrafeRunLength` to make its use clearer?

Reply:
Renamed.